json-guides/retry-timeout.json

```diff
@@ -119,7 +119,7 @@
                 "Microprofile Fault Tolerance allows microservices to handle unavailable services. It uses different policies such as Timeout and Retry to guide the execution and result of some logic. The MicroProfile Fault Tolerance 1.0 feature provides an environment to support resilient microservices through patterns that include timeout and retry. Enable the MicroProfile Fault Tolerance 1.0 feature in the <code>server.xml</code> file of the Open Liberty server."
             ],
             "instruction": [
-                "Add the following element declaration inside the <code>featureManager</code> element of the <code>server.xml</code> file, or click <action title='Enable MicroProfile Fault Tolerance' onclick=\"retryTimeoutCallback.addMicroProfileFaultToleranceFeatureButton(event)\">&lt;feature>mpFaultTolerance-1.0&lt;/feature&gt;</action>.<br>Then, click <action title='Run' onclick=\"retryTimeoutCallback.saveServerXMLButton(event)\">Run</action> on the editor menu pane."
+                "Add the following element declaration that is inside the <code>featureManager</code> element of the <code>server.xml</code> file.<br><codeblock>&lt;feature>mpFaultTolerance-1.0&lt;/feature&gt;</codeblock><br>Alternatively, click <action title='Enable MicroProfile Fault Tolerance' onclick=\"retryTimeoutCallback.addMicroProfileFaultToleranceFeatureButton(event)\">Add</action>.<br>Then, click <action title='Run' onclick=\"retryTimeoutCallback.saveServerXMLButton(event)\">Run</action> on the editor menu pane."
```

Comment:
Extra text had been added....please remove:
'Add the following element declaration ~~that is~~ inside the featureManager...'